src/components/Base/Hero.stories.js

```diff
@@ -0,0 +1,89 @@
+import { storiesOf } from "@storybook/vue";
+import { VApp, VContent } from "vuetify/lib"; // <-- add the import
+
+import Hero from "./Hero.vue";
+import yoga2x from '../../assets/yoga@2x.png';
```

Comment:
Also possible to use absolute imports:

```
import yoga2x from '@/assets/yoga@2x.png';
```

Reply:
Never mind - absolute imports can be used in the Vue project itself, but not the Storybook stories